### DISCUSSION TITLE
windows instalation

### DISCUSSION THREAD

**Discussion by bkt1916** [keycloak/keycloak#35982] [Q&A]
Hello, is there any official support for windows instalaltion?

**Comment by bkt1916** [comment]
And maybe more important, is there all options avaiable from the manual where it is written how to go onto production from dev.?

**Comment by ssilvert** [comment]
I'm not sure what you mean by "official support", but we do make sure everything works on Windows.

If you are talking about paid support for RHBK provided by Red Hat, I think the answer is yes.  But you will need to check with a sales associate to make sure.

**Comment by bkt1916** [comment]
BY official documentation i mean - there is no info about any windows config on Your documents. Only one command to run dev env. All rest and all examples i could find on the internet plp are using linux. I need everything on windows:)

**Comment by Xulunix** [comment]
I couldn't find any differences for running on windows vs. linux.
Only difference in windows is how you set environment variables, other than that, everything should work without any changes needed.
